app/top-page.module.css

```
@@ -0,0 +1,362 @@
+  height:100px;
+  background-color: rgb(76, 150, 252);
+}
+.newsTitle {
+  font-size: 2rem;
+  font-family: Yu Mincho;
+  color: rgb(255, 255, 255);
+  margin-left: 5%;
+}
+.event {
+  font-size: 1.5rem;
+  font-family: Yu Mincho;
```

Stylelint (17.9.0)

[error] Expected quotes around "Yu Mincho" (font-family-name-quotes)

(font-family-name-quotes)